Calculator/infix_to_rpn.py

```diff
@@ -2,13 +2,13 @@
 This module contains the functions used by Calculator program
 to change infix notation into postfix notation
 """
-import numbers
+from Calculator.__init__ import _is_number
 output = []
```

Comment:
I know this is not part of the PR, but can you move these variables inside of a function? I don't think they are necessary as global variables.